Packages.props

```diff
@@ -44,5 +44,10 @@
 		<PackageReference Update="Azure.Storage.Blobs" Version="12.10.0" />
 		<PackageReference Update="coverlet.collector" Version="3.1.2" />
 		<PackageReference Update="coverlet.msbuild" Version="3.1.2" />
+
+		<PackageReference Update="Azure.ResourceManager" Version="1.3.1" />
```

Comment:
Why are you adding these other references? They aren't being added in any of the projects so the additions here aren't going to do anything.